Ugly, I know - but so are non dietlibc C libraries ... ,-)

  - Rene Rebe <rene@exactcode.de>

--- mininet/host.c.vanilla	2006-10-04 10:41:45.000000000 +0000
+++ mininet/host.c	2006-10-04 10:44:07.000000000 +0000
@@ -36,6 +36,9 @@
 #include <ip6.h>
 #include <uint32.h>
 
+#undef dn_expand
+#undef res_query
+
 typedef struct { const char* name; unsigned short value; } rec;
 
 rec classes[] =
--- mininet/arping.c.vanilla	2006-10-04 11:15:31.000000000 +0000
+++ mininet/arping.c	2006-10-04 11:15:36.000000000 +0000
@@ -2,7 +2,7 @@
 #include <inttypes.h>
 #include <sys/socket.h>
 #include <netpacket/packet.h>
-#include <net/if_ether.h>
+#include <linux/if_ether.h>
 #include <arpa/inet.h>
 #include <sys/poll.h>
 #include <unistd.h>
--- mininet/tcpdump.c.vanilla	2006-10-04 11:15:10.000000000 +0000
+++ mininet/tcpdump.c	2006-10-04 11:15:14.000000000 +0000
@@ -3,7 +3,7 @@
 
 #include <sys/socket.h>
 #include <net/if.h>
-#include <net/if_ether.h>
+#include <linux/if_ether.h>
 #include <net/if_arp.h>
 #include <netpacket/packet.h>
 #include <unistd.h>
--- mininet/tstat.c.vanilla	2006-10-04 11:15:43.000000000 +0000
+++ mininet/tstat.c	2006-10-04 11:15:47.000000000 +0000
@@ -4,7 +4,7 @@
 #include <stdlib.h>
 #include <sys/socket.h>
 #include <net/if.h>
-#include <net/if_ether.h>
+#include <linux/if_ether.h>
 #include <net/if_arp.h>
 #include <netpacket/packet.h>
 #include <unistd.h>
